command-parser.js

```diff
@@ -44,9 +44,13 @@ if (customCommands && customCommands.commands) {
 }
 
 // Install plug-in commands
-
+var plugins = exports.plugins = {};
 fs.readdirSync('./chat-plugins').forEach(function (file) {
-	if (file.substr(-3) === '.js') Object.merge(commands, require('./chat-plugins/' + file).commands);
+	if (file.substr(-3) === '.js') {
+		var plugin = require('./chat-plugins/' + file);
+		Object.merge(commands, plugin.commands);
+		if (Object.keys(plugin).length > 1) plugins[file.slice(0, -3)] = Object.reject(plugin, 'commands');
```

Comment:
This checks if there are other properties than ``commands`` exported by a plugin, then adds those to ``CommandParser.plugins`` under the name of the script.